dlt-user/src/lib.rs

```
@@ -209,16 +213,18 @@ pub struct DltUserInner {
     receiver: channel::Receiver<Message>,
     sender: channel::Sender<Message>,
     mainloop_joinhandle: Option<JoinHandle<()>>,
-    rb : HeapRb<u8>,
+    rb: HeapRb<u8>,
+    user_path: Option<PathBuf>,
 }
 
-// impl Drop for DltUserInner{
-//     fn drop(&mut self) {
-//         // let file_path = self.dlt_user_handle.unwrap().;
-//         // fs::remove_file(self.dlt_log_handle.unwrap().path());
-//         // unsafe { libc::unlink(user_path.as_ptr() as *const i8) };
-//     }
-// }
+impl Drop for DltUserInner {
+    fn drop(&mut self) {
+        if let Some(_path) = self.user_path.as_ref() {
+            if let Ok(()) = fs::remove_file(self.user_path.as_ref().unwrap()) {};
```

unwrap here could cause a panic on drop. Avoid using unwrap here. Just ignore any errors.